packages/generator/templates/app/app/auth/mutations/forgotPassword.test.ts

```diff
@@ -41,7 +41,7 @@ describe("forgotPassword mutation", () => {
     await forgotPassword({ email: user.email }, {} as Ctx)
 
     const tokens = await db.token.findMany({ where: { userId: user.id } })
-    const token = tokens[0]
+    const token = tokens[0]!
```

Comment:
Thanks @benbender!

Instead of adding `!`, let's add an if statement that throws an error if it doesn't exist. That will fix the type error and help debug a runtime issue.